README.md

```diff
@@ -1,13 +1,90 @@
+![image or gif](link in the parens) - I will work on this in the morning!
 
-## Set Up
+[Make your own poster!](https://pcmueller.github.io/hang-in-there-boilerplate/)
 
-1. One teammate: fork this repository
-2. Go to settings and turn on GitHub Pages for this repository
-3. All teammates: clone down this repository
-4. `cd` into the repository
-5. Run `open index.html` to view it in the browser
+#### Overview & [Project](https://frontend.turing.io/projects/module-1/hang-in-there.html) Goals
+- Sometimes you need a pick me up. Remember those motivational posters that were all over the place in classrooms and online? We’re going to make our own!
+- Write clean, DRY JavaScript to store our data
+- Use a provided class by creating object instances using the new keyword
+- Manipulate the page after it has loaded adding, removing, and updating elements on the DOM
+- Begin to understand the connection between HTML, CSS and JavaScript
 
-Project spec & rubric can be found [here](https://frontend.turing.io/projects/module-1/hang-in-there.html)
+#### Learning Goals
+- Develop a better understanding of learned concepts
+- Work on teaching strategies to grow as a developer
+- Get more reps and add to toolbox and tying concepts together
+- Work on teamwork and remote collaboration skills
+
+#### Why was this project created?
+- This project was created to teach the fundamentals of working on a team and implementing vanilla JavaScript to create a basic web application
+
+#### How does this work?
+![homepage screenshot](link in the parens)
+- Each page load will automatically generate a random motivational poster. From here, the user has options!
+- The user may continue to generate random posters by clicking the `Show Another Random Poster Button`.
+- To create their own poster, a user will click on `Make Your Own Poster`. This will pull up a form that accepts inputs for an image URL, a title, and a quote. Clicking `Show my poster` will return the user to the home page and display their new poster! There is also an option to return to the home page without creating a poster, by clicking `Nevermind, take me back!`
+- Posters can also be stored for later viewing by clicking `Save This Poster`. This switches to a grid view displaying mini versions of all saved posters.
+![saved posters view + delete gif](link in the parens)
+- To delete a poster, the user can double click the image.
+- This applications runs directly in the web browser!
+
+#### Want to contribute?
+- Interested in contributing to Hang in there?!
+- Fork this project and clone to your machine.
+- Create a pull request, and one of our project managers will get back to you with a code review!
+
+#### Technologies Used
+- JavaScript
+- HTML
+- CSS
+
+#### Project Reflection
+##### Nichele
+- I had a great experience during this project. Peter and I spent a good deal of time planning and discussing the problems, challenges we might face, and how to address problems when encountered.
+##### Peter
+
```

Comment:
"On reflection, I feel very similarly.  This was a successful and enjoyable experience overall, primarily driven by strong communication, a patient approach to planning and debugging, and a commitment to writing code together."

Reply:
√